arm/Microsoft.RecoveryServices/vaults/backupStorageConfig/deploy.bicep

```diff
@@ -0,0 +1,34 @@
+@description('Required. Name of the Azure Recovery Service Vault')
+@minLength(1)
+param recoveryVaultName string
+
+@description('Optional. Change Vault Storage Type (Works if vault has not registered any backup instance)')
+@allowed([
+  'GeoRedundant'
+  'LocallyRedundant'
+  'ReadAccessGeoZoneRedundant'
+  'ZoneRedundant'
+])
+param vaultStorageType string = 'GeoRedundant'
+
+@description('Optional. Enable CRR (Works if vault has not registered any backup instance. Not compatible with Locally Redundant Storage (LRS))')
+param enableCRR bool = true
```

Comment:
Please replace with `crossRegionRestoreFlag` to reflect official reference and update description 
Ref https://docs.microsoft.com/en-us/azure/templates/microsoft.recoveryservices/vaults/backupstorageconfig?tabs=bicep